source/wp-content/themes/wporg-news-2021/sass/post/_single.scss

```diff
@@ -0,0 +1,109 @@
+body.single {
+
+  .entry-header {
+    padding: var(--wp--custom--alignment--edge-spacing);
+
+    @include break-large() {
+      display: flex;
+      flex-direction: row;
+      gap: var(--wp--custom--alignment--edge-spacing);
+    }
+
+    .entry-meta {
+      padding: 18px 0;
+      flex-direction: row;
+      gap: 12px;
+
+      @include break-large() {
+        width: 33vw;
+        gap: 0;
+        flex-direction: column;
+      }
+
+      .wp-block-post-date {}
+
+      .author-and-terms {
+        display: flex;
+        flex-direction: row;
+        gap: 12px;
+      }
+
+      .wp-block-post-author {
+        font-size: var(--wp--preset--font-size--small);
+
+        // Lets just pretend this markup doesn't exist...
+        .wp-block-post-author__content,
+        .wp-block-post-author__name {
+          display: contents;
+        }
+      }
+
+      .wp-block-post-terms {
+        text-transform: none;
+
+        &:after {
+          display: none;
+        }
+      }
+    }
+
+    .wp-block-post-title {
+      font-size: 60px;
+      line-height: 1.1;
+
+      @include break-large() {
+        font-size: 70px;
+        line-height: 1.3;
+        flex-grow: 1;
+        min-width: var(--wp--custom--layout--content-size);
+      }
+    }
+  }
+
+  // A container for the post-content block.
+  .wp-block-group.content-section {
+    display: flex;
+    flex-direction: column;
+    padding: var(--wp--custom--alignment--edge-spacing);
+
+    @include break-large() {
+      flex-direction: row;
+      gap: var(--wp--custom--alignment--edge-spacing);
+
+      // Create a psuedo-element to offest the post content on
+      // larger viewports.
+      &:before {
+        content: '';
+        display: block;
+        height: 100px;
+        width: 33vw;
+      }
+    }
+  }
+
+  // The _actual_ post-content block.
+  .wp-block-post-content {
+
+    @include break-large() {
+      min-width: var(--wp--custom--layout--content-size);
+    }
+
+    // We removed the default block gap in the normalize.scss. This
+    // brings it back within the post-content area.
+    *[class*="wp-block-"],
+    p,
+    & > h1,
+    & > h2,
+    & > h3,
+    & > h4,
+    & > h5,
+    & > h6 {
+      margin-top: var(--wp--style--block-gap);
```

Comment:
This is putting a margin on subsequent columns in a columns block:
<img width="751" alt="Screen Shot 2022-01-20 at 10 50 11 AM" src="https://user-images.githubusercontent.com/541093/150381875-82689374-ac41-4b95-896f-77967fe43d8b.png">